project/src/main/java/edu/depaul/cdm/se452/project/model/Student.java

```diff
@@ -27,7 +27,7 @@
 
 	@Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private int studentId;
+    private int student_id;

```

Comment:
Why are the changes needed for modifying the Student member variable names?  Is this simply to make the SQL work properly? If so, the SQL query you are using should be using student_id and this can remain studentId as Flyway recognizes camel case and automatically converts the column names to include the underscore (such as student_id).

Reply:
This would not work for me i tried everything i'm not sure